docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -81,6 +81,20 @@ To learn more about the ``ansible-galaxy`` command-line tool, see the :ref:`ansi
 
 .. _creating_collection_skeletons:
 
+Creating collections with ansible-creator
+=========================================
+
+`ansible-creator <https://github.com/ansible/ansible-creator/>`_ is designed to quickly scaffold an Ansible collection project.
+
+.. note::
+
+   Installing ansible-creator using the `Ansible Development Tools <https://github.com/ansible/ansible-dev-tools>`_ package is recommended.
+
+After `installing <https://ansible.readthedocs.io/projects/creator/installing/#installation>`_ ``ansible-creator`` you can initialize a project in one of the following ways:
+
+* Use the `init <https://ansible.readthedocs.io/projects/creator/installing/#initialize-ansible-collection-init-subcommand>`_ subcommand.
+* To interact with the GUI, `setup <https://ansible.readthedocs.io/projects/creator/collection_creation/#step-1-installing-ansible-creator-in-the-environment>`_ Visual Studio Code and the Ansible extension, then open VS Code in an environment/project where you installed ``ansible-creator`` and then follow the instructions in the Ansible extension.
```

Comment:
A few too many 'thens.' Rather than try to cover it all, I'd suggest just pointing to the vscode setup.  
Also, for bullet lists, as @oraNod mentioned (I think in a different PR?) we want to use parallelism... that is, each bullet item should start the same way, and usually something active. So the first one is 'Use the 'init'...'  That suggests the second one should similarly start with a verb.  It could be 'Use `ansible-creator` within Visual Studio Code and the Ansible extension.'  And then make that the link you use above.